packages/app/components/claim/claim-form.tsx

```diff
@@ -401,16 +405,8 @@ export const ClaimForm = ({
             <Button
               size="regular"
               variant="primary"
-              disabled={
-                state.status === "loading" ||
-                (edition.gating_type === "multi" && !location)
-              }
-              tw={
-                state.status === "loading" ||
-                (edition.gating_type === "multi" && !location)
-                  ? "opacity-[0.45]"
-                  : ""
-              }
+              disabled={isDisableButton}
+              tw={isDisableButton ? "opacity-[0.45]" : ""}
```

Comment:
IMO reducing opacity should be an inner button feature when `disabled`  instead of this, I'm happy to do this.